Cargo.toml

```diff
@@ -43,7 +45,7 @@ default = ["default-tls", "tokio", "ureq"]
 # These features are only relevant when used with the `tokio` feature, but this might change in the future.
 default-tls = []
 native-tls = ["dep:reqwest", "reqwest/default", "dep:native-tls", "ureq/native-tls"]
-rustls-tls = ["dep:rustls", "reqwest/rustls-tls"]
+rustls-tls = ["dep:reqwest", "dep:rustls", "reqwest/rustls-tls"]
```

Comment:
I also added back the `dep:reqwest` to the `rustls-tls` feature, as it didn't make sense to me _not_ to depend on this crate in the `rustls-tls` case. LMK if I missed something.